manufacturing-ai-suite/industrial-edge-insights-multimodal/docs/user-guide/index.rst

```diff
@@ -24,5 +27,6 @@ operational efficiency.
    get-started
    how-to-build-from-source
    how-to-configure-alerts
+   how-to-deploy-with-helm

```

Comment:
This already existed. Now there are two of the same file in the TOC:
https://github.com/open-edge-platform/edge-ai-suites/blob/main/manufacturing-ai-suite/industrial-edge-insights-multimodal/docs/user-guide/index.rst?plain=1#L29

```
   how-to-deploy-with-helm
   how-to-configure-alerts
   how-to-deploy-with-helm
```

Reply:
Do better.